.github/workflows/_xpu-test.yml

```diff
@@ -47,6 +47,10 @@ on:
         type: boolean
         default: true
 
+permissions:
+  id-token: write
```

Comment:
I remember the discussion, that OIDC requires id-token, but does it have to be with write permissions? Would be nice to put a link to some sort of public document in PR description explaining why this is necessary

Reply:
Yes, OIDC requires `id-token: write` in order to write the JWT ID token for the workflow to access. It is documented here https://docs.github.com/en/actions/security-for-github-actions/security-hardening-your-deployments/configuring-openid-connect-in-cloud-providers#adding-permissions-settings